### ISSUE TITLE
Is it possible to compress already existing trained network?

### ISSUE THREAD

**Issue by mrgloom** [senya-ashukha/variational-dropout-sparsifies-dnn#8]
Is it possible to compress already existing trained network?

**Comment by senya-ashukha** [comment]
Hi, 

at least you can try :) You just need to init mean values of your weights with you pre-trained network,  init log_variances^2 with something really small like -8 and add the regularizer from the paper, it probably should works.  Empirically we see that if your existing network is trained with l2 it works better.
